#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_mp4ff-static_makefile.dpatch by  <matthew@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add a static build of mp4ff to Makefile

@DPATCH@
diff -urNad faad2-svn~/common/mp4ff/Makefile.am faad2-svn/common/mp4ff/Makefile.am
--- faad2-svn~/common/mp4ff/Makefile.am	2007-11-28 23:34:18.000000000 +0000
+++ faad2-svn/common/mp4ff/Makefile.am	2007-11-28 23:40:29.000000000 +0000
@@ -1,8 +1,9 @@
-noinst_LTLIBRARIES = libmp4ff.la
+lib_LIBRARIES = libmp4ff.a
+include_HEADERS = mp4ff.h mp4ffint.h
 
-libmp4ff_la_CFLAGS = -DUSE_TAGGING=1
+libmp4ff_a_CFLAGS = -DUSE_TAGGING=1
 
-libmp4ff_la_SOURCES = mp4ff.c mp4atom.c mp4meta.c mp4sample.c mp4util.c \
-   mp4tagupdate.c mp4ff.h mp4ffint.h mp4ff_int_types.h \
-   drms.h drms.c drmstables.h
+libmp4ff_a_SOURCES = mp4ff.c mp4atom.c mp4meta.c mp4sample.c mp4util.c \
+		     mp4tagupdate.c mp4ff.h mp4ffint.h mp4ff_int_types.h \
+		     drms.h drms.c drmstables.h
 
diff -urNad faad2-svn~/frontend/Makefile.am faad2-svn/frontend/Makefile.am
--- faad2-svn~/frontend/Makefile.am	2007-11-28 23:34:18.000000000 +0000
+++ faad2-svn/frontend/Makefile.am	2007-11-28 23:39:28.000000000 +0000
@@ -1,11 +1,11 @@
 bin_PROGRAMS = faad
 
 INCLUDES = -I$(top_srcdir)/include -I$(top_srcdir)/common/faad \
-   -I$(top_srcdir)/common/mp4ff
+	   -I$(top_srcdir)/common/mp4ff
 
 faad_LDADD = $(top_builddir)/libfaad/libfaad.la \
-	$(top_builddir)/common/mp4ff/libmp4ff.la
+	     $(top_builddir)/common/mp4ff/libmp4ff.a
 
 faad_SOURCES = main.c \
-	audio.c audio.h \
-	$(top_srcdir)/common/faad/getopt.c
+	       audio.c audio.h \
+	       $(top_srcdir)/common/faad/getopt.c
diff -urNad faad2-svn~/plugins/xmms/src/Makefile.am faad2-svn/plugins/xmms/src/Makefile.am
--- faad2-svn~/plugins/xmms/src/Makefile.am	2007-11-28 23:38:02.000000000 +0000
+++ faad2-svn/plugins/xmms/src/Makefile.am	2007-11-28 23:38:02.000000000 +0000
@@ -8,7 +8,7 @@
        -I$(top_srcdir)/include -I$(top_srcdir)/common/mp4ff
 
 libmp4_la_LIBADD = $(top_builddir)/libfaad/libfaad.la \
-       $(top_builddir)/common/mp4ff/libmp4ff.la
+       $(top_builddir)/common/mp4ff/libmp4ff.a
 
 libmp4_la_LDFLAGS = -module -avoid-version $(local_LDFLAGS) -lpthread
 
